### PR TITLE
Add pronouns to People list and Profile

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-13762
affects: student, teacher, parent
release note: none

Test plan:
* narmstrong > s > CS 1400 > People > Student1 Armstrong should have pronouns
* Open Profile drawer, should see pronouns

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1018806&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student, Teacher, Parent

[MBL-13762](https://instructure.atlassian.net/browse/MBL-13762)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.44% | 93.42% | 0.02%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Profile/ProfileViewController.swift [hunk 1]
```diff
@@ -108,6 +101,16 @@ public class ProfileViewController: UIViewController, ProfileViewProtocol {
...
         tableView?.reloadData()
+        reloadProfile()
+    }
+
+    func reloadProfile() {
+        let profile = presenter?.profile.first
+        let userName = presenter?.profile.first?.name ?? env.currentSession?.userName
+        avatarView?.name = userName ?? ""
+        avatarView?.url = profile?.avatarURL
+        nameLabel?.text = userName.flatMap { User.displayName($0, pronouns: profile?.pronouns) }
```
Comment (vanwagonet) [inline Core/Core/Profile/ProfileViewController.swift]: do we really need pronouns in the "my profile"? I thought they are mostly for helping you politely refer to other people.

**Pull request merged**